docs/content/en/highlights/2021-07-23-vrl-match_datadog_query.md

```diff
@@ -0,0 +1,133 @@
+---
+date: "2021-07-23"
+title: "Datadog Search Syntax support"
+description: "Use Datadog Search Syntax in `filter` transforms and VRL"
+authors: ["leebenson"]
+pr_numbers: [7837, 8370],
+release: "0.16.0"
+hide_on_release_notes: false
+badges:
+  type: new feature
+  providers: ["datadog"]
+  domains: ["vrl", "filter transform"]
+---
+
+This release adds support for [Datadog Search Syntax](datadog_seach_syntax), a
+query language based loosely on [Lucene](lucene) and designed for parsing
```

Comment:
```suggestion
query language based loosely on [Lucene][] and designed for parsing
```

Luc pointed out to me recently that `hugo` will automatically pick these up if the text is the same as the footer link (case insensitive).